application/actions/organization_fixtures_test.go

```diff
@@ -238,14 +239,57 @@ func fixturesForOrganizationDomain(as *ActionSuite) OrganizationFixtures {
 }
 
 func fixturesForUpdateOrganization(as *ActionSuite) OrganizationFixtures {
-	var org models.Organization
-	org = models.Organization{
-		Name:       "default org",
-		AuthType:   models.AuthTypeSaml,
-		AuthConfig: "{}",
-		Url:        nulls.NewString("https://www.example.com"),
+	orgs := make([]models.Organization, 3)
+	orgs[0].Name = "default org"
+	orgs[1].Name = "trusted org 1"
+	orgs[2].Name = "trusted org 2"
+	for i := range orgs {
+		orgs[i].AuthType = models.AuthTypeSaml
+		orgs[i].AuthConfig = "{}"
+		orgs[i].Url = nulls.NewString("https://www.example.com")
+		test.MustCreate(as.DB, &orgs[i])
+	}
+
+	domains := make([]models.OrganizationDomain, 2)
+	for i := range domains {
+		domains[i] = models.OrganizationDomain{
+			OrganizationID: orgs[0].ID,
+			Domain:         strconv.Itoa(i) + orgs[0].UUID.String() + ".example.com",
+		}
+		test.MustCreate(as.DB, &domains[i])
+	}
+
+	userFixtures := test.CreateUserFixtures(as.DB, 1)
+	users := userFixtures.Users
+
+	users[0].AdminRole = models.UserAdminRoleSuperAdmin
+	as.NoError(as.DB.Save(&users[0]))
+
+	var file models.File
+	as.Nil(file.Store("photo.gif", []byte("GIF89a")), "unexpected error storing file")
+
+	return OrganizationFixtures{
+		Users:               users,
+		Organizations:       orgs,
+		File:                file,
+		OrganizationDomains: domains,
 	}
-	test.MustCreate(as.DB, &org)
+}
+
+func fixturesForCreateTrust(as *ActionSuite) OrganizationFixtures {
+	orgs := make([]models.Organization, 3)
+	orgs[0].Name = "default org"
+	orgs[1].Name = "trusted org 1"
+	orgs[2].Name = "trusted org 2"
+	for i := range orgs {
+		orgs[i].AuthType = models.AuthTypeSaml
+		orgs[i].AuthConfig = "{}"
+		orgs[i].Url = nulls.NewString("https://www.example.com")
+		test.MustCreate(as.DB, &orgs[i])
+	}
+
+	trust := models.Trust{PrimaryID: orgs[0].ID, SecondaryID: orgs[1].ID}
```

Comment:
The mirror trust should get created too, right? (If so, I guess your desired test results may change.)

Reply:
This creates both.